userspace/libsinsp/container_engine/docker_common.cpp

```diff
@@ -319,18 +320,25 @@ bool docker_async_source::parse_docker(std::string &container_id, sinsp_containe
 			Json::Value img_root;
 			if(reader.parse(img_json, img_root))
 			{
+			    // img_root["RepoDigests"] contains only digests for images pulled from registries.
+			    // If an image gets retagged and is never pushed to any registry, we will not find
+			    // that entry in container->m_imagerepo. Also, for locally built images we have the
+			    // same issue. This leads to container->m_imagedigest being empty as well.
+			    unordered_set<std::string> imageDigestSet;
```

Comment:
nit: indentation

Reply:
Done. Sorry I was using intellij and it didn't show up easily.